package.json

```diff
@@ -102,6 +102,7 @@
     "build-dev": "yarn clean && yarn build-nav-data && GATSBY_ACTIVE_ENV=development gatsby build",
     "build-staging": "yarn build-nav-data && GATSBY_ACTIVE_ENV=staging gatsby build --prefix-paths",
     "build-prod": "yarn build-nav-data && GATSBY_ACTIVE_ENV=production gatsby build",
+    "build-release": "yarn build-nav-data && GATSBY_ACTIVE_ENV=release gatsby build",
```

Comment:
Remove this - I know you included it for your testing but we don't want to keep this around.